docs/getting-started/getting-started.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Getting Started
+sidebar_position: 0
+---
+
+## What is Golioth?
+
+![Console](./assets/console.png)
+
+Golioth is an IoT platform optimized for the developer experience. We provide a
+single control plane for all of your devices.
+
+### Cloud Side
+
+From over-the-air (OTA) firmare updates, to remote procedure calls (RPC) and
+device fleet settings, to device-to-cloud time series data and bidirectional
+state data, we've already built the management tools you need. Our data-routing
+features ensure your data can go where you need it, whether that's AWS, Azure,
+GCP, or any other server, app, or database platform.
```

Comment:
More of a future thing, but wanted to write it down so I don't forget - maybe we should check with Dylan and Ariana if there are additional/alternative data destinations we want to highlight here (like influx or Mongo)